### PR TITLE
Eng 1205 cluster wide deployment

### PR DESCRIPTION
**Pull request by ampie**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ampie**

If they are not already assigned, you can assign the PR to them by writing `/assign @ampie` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-k8s-controller-coordinator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-k8s-controller-coordinator/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
@ampie: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 61487f6e404e0029aa6ebcba24d64c67f04d6c40 | [link](https://raw.githubusercontent.com/entando-k8s/devops-results/logs/jenkins-x/logs/entando-k8s/entando-k8s-controller-coordinator/PR-41/1.log) | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-k8s-controller-coordinator-carcpr41**

**Comment by entando-jx** [comment]
:star: PR built and available in a preview environment **entando-k8s-controller-coordinator-carcpr41**

File: src/test/java/org/entando/kubernetes/controller/coordinator/inprocesstests/EntandoOperatorMatcherTest.java
Comment (firegloves) [inline src/test/java/org/entando/kubernetes/controller/coordinator/inprocesstests/EntandoOperatorMatcherTest.java:37]: I'm not sure if it is an oversight or not. There is a reason to execute these lines before and after?

Comment (ampie) [reply src/test/java/org/entando/kubernetes/controller/coordinator/inprocesstests/EntandoOperatorMatcherTest.java:37]: Indeed this was intentional. Each test case assumes that the relevant system properties are unset, hence the BeforeEach. But at the same time, we can't leave the system properties there after the test cases, as it will affect the behaviour of the rest of our tests. Hence the AfterEach

**Review by Kerruba — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando-k8s/entando-k8s-controller-coordinator/pull/41#pullrequestreview-483888007" title="Approved">Kerruba</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-k8s-controller-coordinator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando-k8s/entando-k8s-controller-coordinator/blob/master/OWNERS)~~ [Kerruba]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
